packages/frontend/src/components/dialogs/ViewGroup.tsx

```diff
@@ -406,6 +409,14 @@ function ViewGroupInner(
                     quantity: group.contactIds.length,
                   })}
             </div>
+            <div className='group-member-filter'>
+              <SearchInput
```

Comment:
This component cannot be used, because it's specific to the main screen.

https://github.com/deltachat/deltachat-desktop/blob/8ca44bd1bb2ac7212787980654785013be9a2220/packages/frontend/src/components/SearchInput/index.tsx#L48-L57

Reply:
Solved